MBBSEmu/Logging/CustomLogger.cs

```diff
@@ -14,20 +14,12 @@ public class CustomLogger : Logger
         static CustomLogger()
         {
             var config = new NLog.Config.LoggingConfiguration();
-
             var consoleLogger = CreateConsoleTarget();
-            config.AddTarget(consoleLogger);
 
-            var fileLogger = new FileTarget("fileLogger")
-            {
-                FileName = @"c:\dos\log\log.txt",
-                Layout = Layout.FromString("${shortdate} ${time} ${level} ${callsite} ${message}"),
-                DeleteOldFileOnStartup = true
-            };
-            //config.AddTarget(fileLogger);
+            config.AddTarget(consoleLogger);
             config.AddRuleForAllLevels(consoleLogger);
-            //config.AddRuleForAllLevels(fileLogger);
             LogManager.Configuration = config;
+            LogManager.Configuration.Variables["mbbsdir"] = Directory.GetCurrentDirectory() + "\\";
```

Comment:
Use `Path.DirectorySeparatorChar` vs `\\` as this would not work on Linux/OSX

https://docs.microsoft.com/en-us/dotnet/api/system.io.path.directoryseparatorchar?view=net-5.0

Reply:
Or better yet, `Path.Combine` or `Path.Join`

Reply:
I'm adding a filename on to the end of this and it needs a trailing slash. Join/Combine don't seem to address that.